components/ingest-service/server/jobs.go

```diff
@@ -1,124 +1,256 @@
 package server
 
 import (
-	"github.com/chef/automate/components/ingest-service/backend"
-	"github.com/chef/automate/components/ingest-service/config"
+	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
 	log "github.com/sirupsen/logrus"
+	rrule "github.com/teambition/rrule-go"
 	"golang.org/x/net/context"
+
+	"github.com/chef/automate/api/interservice/ingest"
+	"github.com/chef/automate/components/ingest-service/backend"
+	"github.com/chef/automate/components/ingest-service/config"
+	"github.com/chef/automate/lib/cereal"
 )
 
-func (server *JobSchedulerServer) addUpdateJob(jc config.JobConfig) {
-	var cronFunc func()
+func InitializeJobManager(c *cereal.Manager, client backend.Client) error {
+	err := c.RegisterTaskExecutor("delete_expired", &DeleteExpiredMarkedNodesTask{client}, cereal.TaskExecutorOpts{})
```

Comment:
I think it'd be nice for some package to export constants for these task/workflow names and to use the constants everywhere.